### ISSUE TITLE
Web interface failing to load first-time on Vivaldi

### ISSUE THREAD

**Issue by EmeraldSlash** [urbit/urbit#2763]
**Describe the bug**
When I load the web interface for the first time using the Vivaldi browser (chromium based) the page remains blank and I get the following errors in the developer console (problem source seems to be 404?):
![image](https://user-images.githubusercontent.com/25362339/79625518-c12bf500-817d-11ea-9734-cf867e8e8da0.png)
Reloading the page once fixes the issue, and it does not happen again until the next session (e.g. restart browser - opening/closing tabs still works presumably because of caching on either my browser or Urbit). This issue has been happening for at least 5 days, across multiple OTAs, both on my Ubuntu VM and then my $10 DO droplet when I switched. This only happens on Vivaldi desktop, I have Vivaldi on mobile and it works fine.

**To Reproduce**
1. Go to your urbit's website in the Vivaldi browser
2. Observe the empty page and errors in developer console
3. Reload the page and see it load properly

**Desktop (please complete the following information):**
```
Vivaldi | 2.11.1811.52 (Stable channel) (64-bit)
Revision | 88f335a58f46e6372fa8f4d540ccac8380af31a6
OS | Windows 10 OS Version 1903 (Build 18362.778)
JavaScript | V8 8.0.426.30
Flash | 32.0.0.363
```
- Base hash: 0v1e.va6cc.cks0i.8dhed.sre6p.pi67c.4b9he.d9c5s.ii3v2.c2v76.i2ffu

**Comment by matildepark** [comment]
Vivaldi seems to be rather moot to this — it's that channel.js gives 404s. We did move channel.js out of Eyre, and if OTAs didn't complete I could see this occurring? In any case, I found that [doing these instructions](https://github.com/urbit/urbit/issues/2675#issuecomment-610065176) seems to resolve the same issue for others, give this a shot and let's see if this persists.

**Comment by EmeraldSlash** [comment]
That solution didn't work unfortunately!

**Comment by matildepark** [comment]
@loganallenc @ixv Do either of you know why we would get infrequent 404s on an open channel?

**Comment by loganallenc** [comment]
Yes, I see this sometimes at my cabin due to bad internet connection. I don’t know of any way to tweak the EventSource to prevent it.

**Comment by Fang-** [comment]
> I don’t know of any way to tweak the EventSource to prevent it.

Might be an Eyre bug? iirc it cleans up channels if there's nobody listening on them for a small handful of events. Maybe the slowness of the connection is incorrectly triggering that behavior, or something similar?

(Don't know if that's actually the case, but considering we _are_ getting a response, seems more likely than some obscure EventSource issue.)
